src/App.tsx

```diff
@@ -1,25 +1,120 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [newToDo, setNewToDo] = useState({
+    title: '',
+    user: '',
+  });
+  const todosCopy = [...todosFromServer];
+  const copy = [...usersFromServer];
+
+  const [toDoList, setToDoList] = useState(todosCopy);
+
+  function submitHandler(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    const maxId = Math.max(...todosCopy.map(todo => +todo.id));
+    const eventUser = (event.currentTarget[1] as HTMLInputElement).value;
+    const eventTitle = (event.currentTarget[0] as HTMLInputElement).value;
```

Comment:
make sense to destructure event.currentTarget

Reply:
fixed